src/subscription-user/subscription-user.controller.ts

```diff
@@ -28,4 +38,27 @@ export class SubscriptionUserController {
       createSubscriptionUserDto,
     );
   }
+
+  @Patch('/whatsapp')
+  @ApiBearerAuth('access-token')
+  @ApiOperation({
+    description: 'Cancel an active whatsapp subscription',
+  })
+  @UseGuards(FirebaseAuthGuard)
+  async cancelWhatsappSubscription(
```

Comment:
Out of interest, why label this cancel rather than update? What happens if someone wants to resubscribe?

Reply:
The method only allows cancellation - whereas PATCH means update - hence I thought it was clearer to label it as a `cancel` method